app/lang/en.json

```diff
@@ -286,6 +286,19 @@
     "continueToSigningPage": "Continue to signing page",
 
     "signYourLpa": "Sign your LPA",
+    "signYourLpaContent": "<h2 class=\"govuk-heading-m\">Signing the LPA is an important and powerful legal step.</h2> <p class=\"govuk-body\">When you do this you're officially saying that you want {{ .AllAttorneysFullNames }} to be your finance and affairs attorneys.</p> <p class=\"govuk-body\">This means that once the LPA has been registered, {{ .AllAttorneysFirstNames }} will have the legal power to:</p> <ul class=\"govuk-list govuk-list--bullet\"> <li>help you make a decision about your money or property, or make a decision for you if you do not have the mental capacity to make it yourself</li> <li> deal with organisations, like banks and utility companies, on your behalf</li> </ul>",
```

Comment:
I don't think we should be hard-coding "finance and affairs" in here. (And I know I'm not a content person, but what does it mean to be a "finance and affairs attorney"...)

Reply:
Thinking about it, we'll need to switch this content depending on how many attorneys too

Reply:
Ah yeah good point. Will pluralise and pass in the LPA type. 

Also, re: content 🤷